packages/web-components-stencil/package.json

```diff
@@ -41,7 +41,8 @@
     "@stencil/react-output-target": "0.5.3",
     "@stencil/sass": "3.0.12",
     "@types/jest": "29.5.12",
-    "@utrecht/web-component-library-stencil": "1.3.0",
+    "@utrecht/paragraph-css": "1.1.0",
+    "@utrecht/web-component-library-stencil": "2.0.0",
```

Comment:
Update geeft regressie op Button...

Reply:
Bug aangemaakt bij Utrecht:
https://github.com/nl-design-system/utrecht/issues/2399

Reply:
#198 in deze branch mergen? zodat we nu iig `utrecht.document` hebben, zodat de styling weer klopt...

Reply:
#198 is "broken", @AlineNap gaat het in losse branches oplossen.

Reply:
Wat ik nodig heb zit nu in deze branch